x-pack/plugins/security/public/views/management/edit_user/edit_user.js

```diff
@@ -17,7 +17,7 @@ import { render, unmountComponentAtNode } from 'react-dom';
 import { I18nContext } from 'ui/i18n';
 import { getEditUserBreadcrumbs, getCreateUserBreadcrumbs } from '../breadcrumbs';
 
-const renderReact = (elem, httpClient, changeUrl, username) => {
+const renderReact = (elem, httpClient, changeUrl, username) => { // eslint-disable-line no-unused-vars
```

Comment:
It looks like `httpClient` is unused, why wouldn't we just remove this as opposed to disabling linting?

Reply:
I'm disabling lines that don't relate to angular dependencies but I'm happy to address items pointed out in review. Just trying to conserve time.